### ISSUE TITLE
Incorrect interpretations of bends and bugs related to them

### ISSUE THREAD

**Issue by Dima-S-Jr** [musescore/MuseScore#23604]
## Issue Type

General playback bug

Muse Sounds bug

## Description with steps to reproduce

Below, under each individual subtitle, I have described a specific problem. I don't see the point in publishing each of them in a separate issue. 
In this case, I would like to combine the issues related to Muse Sounds and the general playback bug, because it seems to me that the point here is primarily in Muse Score Studio.

### Bends and slides sound like the same thing

Both slide and bend sound the same. This should not be the case. There is a difference between a slide and a bend.

https://github.com/user-attachments/assets/ef9a34cb-0204-4e80-be88-c3808c5e0ab5

This applies to both MS Basic and Muse Sounds. As an example, I demonstrated one of the 4 bending options (Fall, Doit, Plop and Scoop). But all 4 are similar to the slides.

### Incorrect interpretation of the slides

Above, I showed you how the bends sound. So, at the moment, the "slide out" sound more like bends than slides. This is wrong. The sound should change in pitch as soon as the note is struck, and not after some time.

Below I will try to show you how this happens in MS Basic and Muse Sounds.

**MS Basic** 
![Снимок экрана (445)](https://github.com/user-attachments/assets/153f4743-8ebe-4f1f-9abe-80b0f993e2fd)

**Muse Sounds** (the pitch changes only after the duration has been fully sustained)
![Снимок экрана (446)](https://github.com/user-attachments/assets/c81a146f-51b5-4c46-a94f-47764d827c59)

Also, please pay attention to the "slide in". In MS Basic, it sounds just perfect, as it should (before the beat). But then why does this slide sound on the beat in Muse Sounds?

https://github.com/user-attachments/assets/6bddb0c9-3923-4655-a129-01dc712ef270

Another bug is related to the simultaneous addition of the "slide in" and the "slide out" to the note/chord. In this case, "slide out" will not sound (both in MS Basic and in Muse Sounds).

https://github.com/user-attachments/assets/f1e3d3f4-7365-4cf3-8f3d-5472371d1e33

### Glissando bugs

The first thing I want to say is a misconception of the glissando due to the presence of a bend in front of the note. Below I will show you how this happens in MS Basic and Muse Sounds.

**MS Basic**
![Снимок экрана (449)](https://github.com/user-attachments/assets/cbbd4def-fe6e-4a46-8db8-1b4e6099bca8)

**Muse Sounds** 
![Снимок экрана (450)](https://github.com/user-attachments/assets/c6211401-a765-4f5b-ab55-3f4063c5f947)

Second. I don't quite understand how it turns out that Muse Guitar (unlike MS Basic) reproduces any glissando in the portamento style (even though Diatonic, Chromatic, etc. can be specified in the glissando properties), and MS Basic always has the Chromatic style by default. I think it's wrong to deprive users of the ability to play other guitar glissando styles with Muse Sounds (even if it's not necessary). At the same time, it would be nice to set the default portamento style for all guitar parts (you can also include bowed strings and the like here).

https://github.com/user-attachments/assets/6d6c6ec2-01f2-4190-bdff-c5ccf3c7d18e

## What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio version (64-bit): 4.3.2-241630831, revision: github-musescore-musescore-22b46f2

## Regression

I was unable to check

## Operating system

Windows 11

## Checklist

- ✅ This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- ✅ I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues 
- ✅ I have attached all requested files and information to this report 
- ✅ I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Dima-S-Jr** [comment]
In this issue, I have given examples with guitars. But just listen to how it sounds on some other Muse Sounds instruments (listen to MS Basic first, and then Muse Sounds):

https://github.com/user-attachments/assets/f419557c-070b-4860-af20-c8d851ca40d0

https://github.com/user-attachments/assets/2d1ce490-b4bd-4bb0-8571-d90734e33551

Some instruments sound outrageously distorted, some completely ignore slides/bends. In general, I think that major repairs are required to solve this issue, since I do not know exactly what other bugs may be related to slides/bends, and on what other instruments they may not work.

**Comment by AlexNelsonMusic** [comment]
This is still a prominent issue. They also need to add back functionality to change the curve of the gliss

**Comment by bkunda** [comment]
> I don't see the point in publishing each of them in a separate issue.

There is actually value in differentiating different issues, because it's certainly not always the case that they are being produced by the same cause. As such, we may end up fixing _some_ of the issues raised above via a particular method, however we won't be able to resolve this issue in doing so because there remain other unresolved parts. 

For example, we plan now to address this issue: 

> At the same time, it would be nice to set the default portamento style for all guitar parts (you can also include bowed strings and the like here).

...for which we will need to raise a separate issue so we can link to it when we resolve it (even though it's been raised here).

So in short, please always raise problems in separate issues. GitHub also has a mechanism for embedding sub-issues within a parent issue, which would be more helpful than what we have here in the present issue. 

I recognise there are real problems raised above, but since they will likely need different fixes (especially when a problem relates to MuseSounds but not MS Basic/VST), they really do need to be contained in their own discrete issues. To this end, I'll unfortunately need to close this one for now as it contains too many different (albeit quasi-related) problems for us to meaningfully work with.
